mis_builder/models/mis_builder.py

```diff
@@ -0,0 +1,865 @@
+                domain.append(('period_id', 'in', period_ids))
+            if date_from:
+                domain.extend([('date', '>=', date_from),
+                               ('date', '<=', date_to)])
+            return {
+                'name': expr + ' - ' + this.name,
+                'domain': domain,
+                'type': 'ir.actions.act_window',
+                'res_model': 'account.move.line',
+                'views': [[False, 'list'], [False, 'form']],
+                'view_type': 'list',
+                'view_mode': 'list',
+                'target': 'current',
+            }
+        else:
+            return False
+
+    def compute_period_domain(self, cr, uid, period_report, is_end,
+                              is_initial, context=None):
+        period_obj = self.pool['account.period']
+        move_obj = self.pool['account.move']
+        domain_list = []
+        target_move = period_report.report_instance_id.target_move
+        if target_move == 'posted':
+            domain_list.append(('move_id.state', '=', target_move))
+        if not is_end and not is_initial:
+            if period_report.period_from:
+                compute_period_ids = period_obj.build_ctx_periods(
+                    cr, uid, period_report.period_from.id,
+                    period_report.period_to.id)
+                domain_list.extend([('period_id', 'in', compute_period_ids)])
+            else:
+                domain_list.extend([('date', '>=', period_report.date_from),
+                                    ('date', '<=', period_report.date_to)])
+        elif period_report.period_from and period_report.period_to:
+            period_to = period_report.period_to
+            if is_initial:
+                move_id = move_obj.search(
+                    cr, uid, [('period_id.special', '=', False),
+                              ('period_id.date_start', '<',
+                               period_to.date_start)],
+                    order="period_id desc", limit=1, context=context)
+                if move_id:
+                    computed_period_to = move_obj.browse(
+                        cr, uid, move_id[0], context=context).period_id.id
+                else:
+                    computed_period_to = self.pool['account.period'].search(
+                        cr, uid, [('company_id', '=',
+                                   period_report.company_id.id)],
+                        order='date_start desc', limit=1)[0]
+                # Change start period to search correctly period from
+                period_to = period_obj.browse(cr, uid, [computed_period_to],
+                                              context=context)[0]
+            move_id = move_obj.search(
+                cr, uid, [('period_id.special', '=', True),
+                          ('period_id.date_start', '<=',
+                           period_to.date_start)],
+                order="period_id desc", limit=1, context=context)
```

Comment:
Same issue here. Because, it is to compute the period to use in the build_ctx_periods.